psycopg/psycopg/_cursor_base.py

```diff
@@ -467,15 +466,14 @@ def _check_results(self, results: list[PGresult]) -> None:
 
         for res in results:
             status = res.status
-            if status != TUPLES_OK and status != COMMAND_OK and status != EMPTY_QUERY:
+            if status != TUPLES_OK and status != COMMAND_OK and (status != EMPTY_QUERY):
```

Comment:
Unnecessary parentheses added?

Reply:
Indded, thank you for spotting it. They were added by the AST transformation and I committed them by mistake.